packages/wallet-sdk/src/libInfo.ts

```diff
@@ -0,0 +1,2 @@
+export const LIB_VERSION = '4.1.0';
+export const LIB_NAME = '@coinbase/wallet-sdk';
```

Comment:
Does removing the version.ts file break some of the versioning scripts?

Reply:
updated script